### DISCUSSION TITLE
I have two questions about verdaccio. Do you want to consult?

### DISCUSSION THREAD

**Discussion by Dreacatcher** [verdaccio/verdaccio#2639] [Q&A]
There are two problems

1. Verdaccio is local storage. How to select the server configuration in the early stage and how to ensure that the volume of packages in the later stage is increased for capacity expansion? Is there any relevant introduction

2. Can verdaccio provide external APIs for later statistics?

**Comment by juanpicado** [comment]
Hi 👋!

Maybe you can provide some context, I barely understand the first question, but answer to the second one is, does not exist.
